pkgs/servers/passopolis/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, antBuild, fetchgit, git, python }:
+
+let
+  version = "unstable-2016-05-07";
```

Comment:
If you have a better idea how things should be named, please also update the coding conventions. Otherwise it gets inconsistent across reviews.